coax/_core/stochastic_q.py

```diff
@@ -122,11 +130,12 @@ def num_bins(self):
 
     @classmethod
     def example_data(
-            cls, env, value_range, num_bins=51, observation_preprocessor=None,
-            action_preprocessor=None, value_transform=None, batch_size=1, random_seed=None):
+            cls, env, value_range, proba_dist_hparams=dict(num_bins=51, value_range=[-10, 10]),
```

Comment:
Please don't use mutable defaults. These could generate unexpected side-effects, e.g.
```python
def f(d={}):
    return d

x = f()  # Returns: {}
x['foo'] = 'bar'
f()  # Now returns: {'foo': 'bar'}
```
A typical pattern is to default to `None` and to set the default dict internally.

Reply:
I also missed this in the initial refactoring commit but changed it afterwards.